test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -401,6 +402,7 @@ public void beforeTest() throws Throwable
     @After
     public void afterTest() throws Throwable
     {
+        asyncTestSchemaChanges = true;
```

Comment:
rather than do this, and add blocking logic at every test that fails, why not update `org.apache.cassandra.cql3.CQLTester#schemaChange` to allow not saving the result? We do not care about the schema changes in this block, so we can just ask schemaChange to not update `lastSchemaChangeResult`; this would remove all the blocking.

Reply:
just saw that Benjamin had the same feedback =)